posts/2023-09-25-spring-boot-3-instant-on.adoc

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: "Liberty InstantOn 23.0.0.10-beta provides rapid startup of Spring Boot 3.2 applications"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tjwatson
+author_github: https://github.com/tjwatson
+seo-title: Liberty InstantOn 23.0.0.10-beta provides rapid startup of Spring Boot 3.2 applications
+seo-description: Did you know you can containerize your Spring Boot applications to start up in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features? And with little to no refactoring of the application code? Here’s how with Liberty 23.0.0.10-beta…
+blog_description: "Did you know you can containerize your Spring Boot applications to start up in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features? And with little to no refactoring of the application code? Here’s how with Liberty 23.0.0.10-beta…"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= How to containerize your Spring Boot application for rapid startup
+Thomas Watson <https://github.com/tjwatson>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+Did you know you can containerize your Spring Boot applications to start up in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features? And with little to no refactoring of the application code? Here’s how with Liberty 23.0.0.10-beta…
```

Comment:
Do you have any performance figures you could give here yet?

Reply:
Ah it says "up to 10x" later in the post. Could you mention this in the intro too? Show more concretely why they should care.

Reply:
I will add a link to our performance article.

Reply:
I decided to add a link to a performance article in the summary.